services.yaml

```diff
@@ -4029,6 +4060,80 @@ definitions:
             example: "http://hooks.company.com/overops1/P1/hook"
       #jira_settings: <- TBD
       #udf_settings <- TBD
+  ViewAlertsSummary:
+    description: basic alerts info for a single view
+    type: object
+    properties:
+      view_id:
+        type: string
+        description: View ID
+        example: P12
+      view:
+        type: string
+        description: View Name
+        example: "API"
+      categories:
+        type: array
+        description: Categories Names
+        items:
+          type: string
+        example: "Applications"
+      alerts:
+        type: object
+        properties:
+          global:
+            type: boolean
+            description: Indicates alerts which are global, i.e. will affect all users that share this view.
+            example: false
+          alert_on_new:
```

Comment:
There's also a new boolean param called `alert_on_every_entrypoint` which is true if alert is for every new event regardless of it's entry point.

Standard behavior is a single alert per entrypoint.